net-libs/libwebsockets/libwebsockets-4.0.4.ebuild

```diff
@@ -32,10 +34,12 @@ RDEPEND="
 	libuv? ( dev-libs/libuv:= )
 	sqlite3? ( dev-db/sqlite:= )
 	ssl? (
-		!libressl? ( dev-libs/openssl:0= )
-		libressl?  ( dev-libs/libressl:0= )
-	)
-"
+		!libressl? (
+			!mbedtls? ( dev-libs/openssl:0= )
+			mbedtls? ( >net-libs/mbedtls-2.0.0 )
+			)
```

Comment:
Indentation

Reply:
Fixed in https://github.com/gentoo/gentoo/commit/9887c317408db7527a8dc215b776f489ae1125aa